src/frontend/src/lib/components/ui/ImgBanner.svelte

```diff
@@ -4,11 +4,13 @@
 	export let src: string;
 	export let alt = '';
 	export let width: string | undefined = undefined;
+	export let height: string | undefined = undefined;
+	export let fitHeight = false;
 	export let size: 'small' | 'big' = 'big';
 </script>
 
 <div class="relative block" class:h-56={size === 'big'} class:h-40={size === 'small'}>
-	<div class="absolute w-full">
-		<Img {src} {alt} {width} />
+	<div class="absolute h-full w-full">
```

Comment:
Did you double checked that if the network is slow the change does not result in elements shift in the UI? Asking just to be sure everything remains still in place.

Reply:
yup

Uploading Bildschirmaufnahme 2024-10-23 um 14.46.20.mov…